MODULE.bazel

```diff
@@ -546,7 +546,3 @@ use_repo(
     "busybox",
     "busybox_linux_amd64",
 )
-
-register_toolchains(
-    "//toolchains:ubuntu_cc_toolchain",
-)
```

Comment:
Let's keep `register_toolchains` in 2 places: `WORKSPACE` for our existing workflow and `MODULE.bazel` for future bzlmod workflow.

The plan is to remove `WORKSPACE.bzlmod` when we manage to migrate away from the remaining dependencies.

Reply:
Can't do it; see comment below.